apps/OboeTester/app/src/main/java/com/mobileer/oboetester/WaveformView.java

```diff
@@ -78,7 +78,8 @@ protected void onSizeChanged(int w, int h, int oldw, int oldh) {
         mCurrentWidth = w;
         mCurrentHeight = h;
         mOffsetY = 0.5f * h;
```

Comment:
Should this be "mOffsetY = 0.475f * h"?

Reply:
No, because we want the scaled plot to still be centered between the top and bottom of the viewport.